fireapi/firedb.py

```diff
@@ -16,7 +16,7 @@
 from typing import List, Optional, Union
 from datetime import datetime
 
-DEBUG = True
+DEBUG = False
```

Comment:
This was probably not so smart...

Reply:
no? but https://github.com/Septima/fikspunktsregister/issues/60 ?